docs/install_guides/installing-openvino-conda.md

```diff
@@ -1,10 +1,96 @@
-# Install OpenVINO™ Runtime from Anaconda Cloud
+# Install OpenVINO™ Runtime from Conda Forge {#openvino_docs_install_guides_installing_openvino_conda}
 
-* [Install OpenVINO Runtime from an Archive File](installing-openvino-from-archive-linux.md)
-* [Install OpenVINO from PyPI](installing-openvino-pip.md)
-* [Install OpenVINO with Docker](installing-openvino-docker-linux.md)
-* [Build From Source](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md)
+@sphinxdirective
 
-The other installation methods are temporarily unavailable.
+.. note::
+
+   Installing OpenVINO Runtime from Conda Forge is recommended for C++ developers, as it provides only the C++ Runtime API.
+   If you work with Python, consider :doc:`installing OpenVINO from PyPI <openvino_docs_install_guides_installing_openvino_pip>`
+
+   The current Anaconda package does not provide support for GPU inference.
+
+.. tab:: System Requirements
+
+  | Full requirement listing is available in:
+  | `System Requirements Page <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/system-requirements.html>`__
+
+.. comment to publish in the future, when gpu support comes back:
+
+   .. tab:: Processor Notes
+   
+        | Not all Intel CPUs include integrated graphics processors.  
+        | See `Product Specifications <https://ark.intel.com/>`__ for information about your hardware.
+
+.. tab:: Software
+
+   There are many ways to work with Conda. Before you proceed, learn more about it on the
+   `Anaconda distribution page <https://www.anaconda.com/products/individual/>`__
+
+
+Installing OpenVINO Runtime with Anaconda Package Manager
+############################################################
+
+1.  Set up the Anaconda environment (Python 3.10 used as an example):
+   
+   .. code-block:: sh
+
+      conda create --name py310 python=3.10
+
+   .. code-block:: sh
+
+      conda activate py310
+
+2. Update it to the latest version:
+   
+   .. code-block:: sh
+
+      conda update --all
+
+3. Install the OpenVINO Runtime package:
+
+   .. code-block:: sh
+
+      conda install -c conda-forge openvino=2022.3.0
```

Comment:
BTW, since this guide is for current master, should we have 2023.0.0 version here?

Reply:
I'd stay with 22.3 to keep it functional in nightly... As soon as we get the 23.0 branch, we'll change it there, so that it's ready for the release.